hawkbit-dmf/hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpMessageHandlerService.java

```diff
@@ -373,4 +409,13 @@ private static UpdateMode getUpdateMode(final DmfAttributeUpdate update) {
         return null;
     }
 
-}
+    private boolean isMultiAssignmentsEnabled() {
+        return getConfigValue(MULTI_ASSIGNMENTS_ENABLED, Boolean.class);
+    }
+
+    private <T extends Serializable> T getConfigValue(final String key, final Class<T> valueType) {
+        return systemSecurityContext
+                .runAsSystem(() -> tenantConfigurationManagement.getConfigurationValue(key, valueType).getValue());
```

Comment:
Is this using the right tenant?

Reply:
I don't fully understand why, but some of the tests break if I do not use runAsSystem